src/index.html

```diff
@@ -13,5 +13,93 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <h2>Personal information</h2>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset> <legend>Personal information</legend>
+
+<label>
+  Surname: 
+<input type="text" name="surname" autocomplete="off">  
+</label>
```

Comment:
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
    >
      <fieldset> <legend>Personal information</legend>

      <label>
        Surname: 
        <input 
          type="text" 
          name="surname" 
          autocomplete="off"
        >  
     </label>
```
Please fix all cases
![image](https://user-images.githubusercontent.com/96409563/221654260-4fcca237-cdc7-4a4b-927c-cbb2a206e886.png)
![image](https://user-images.githubusercontent.com/96409563/221654286-0cd1b650-f75e-4a2f-8258-1cad0f3d7417.png)